pylot/simulation/carla_operator.py

```diff
@@ -434,7 +434,8 @@ def __publish_hero_vehicle_data(self, timestamp, watermark_msg):
         velocity_vector = pylot.utils.Vector3D.from_carla_vector(
             self._driving_vehicle.get_velocity())
         forward_speed = velocity_vector.magnitude()
-        can_bus = pylot.utils.CanBus(vec_transform, forward_speed)
+        can_bus = pylot.utils.CanBus(vec_transform, forward_speed,
```

Comment:
Can you please also update https://github.com/erdos-project/pylot/blob/master/pylot/simulation/challenge/ERDOSAgent.py#L196 and https://github.com/erdos-project/pylot/blob/master/pylot/simulation/challenge/ERDOSTrack4Agent.py#L144 ?

`data['linear_velocity'] conains the linear velocity in vehicle reference frame.

Reply:
Does data contain the `velocity_vector`?

Reply:
Resolved: using the orientation (yaw) and forward speed to compute the velocity vector.

Reply:
Done!